[2024-09-21T18:20:55.933+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-21T18:20:55.960+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_table_dag.echo_task scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-21T18:20:55.976+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_table_dag.echo_task scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-21T18:20:55.977+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 3
[2024-09-21T18:20:56.086+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): echo_task> on 2024-09-20 00:00:00+00:00
[2024-09-21T18:20:56.125+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=159) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-21T18:20:56.157+0000] {standard_task_runner.py:72} INFO - Started process 163 to run task
[2024-09-21T18:20:56.151+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'create_table_dag', 'echo_task', 'scheduled__2024-09-20T00:00:00+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/create_table_dag.py', '--cfg-path', '/tmp/tmpwica90vn']
[2024-09-21T18:20:56.161+0000] {standard_task_runner.py:105} INFO - Job 179: Subtask echo_task
[2024-09-21T18:20:56.501+0000] {task_command.py:467} INFO - Running <TaskInstance: create_table_dag.echo_task scheduled__2024-09-20T00:00:00+00:00 [running]> on host 6e2558f7c62d
[2024-09-21T18:20:57.091+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='create_table_dag' AIRFLOW_CTX_TASK_ID='echo_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T00:00:00+00:00'
[2024-09-21T18:20:57.123+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-21T18:20:57.174+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-21T18:20:57.184+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo created table successfully']
[2024-09-21T18:20:57.328+0000] {subprocess.py:86} INFO - Output:
[2024-09-21T18:20:57.353+0000] {subprocess.py:93} INFO - created table successfully
[2024-09-21T18:20:57.356+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-21T18:20:57.516+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-21T18:20:57.527+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=create_table_dag, task_id=echo_task, run_id=scheduled__2024-09-20T00:00:00+00:00, execution_date=20240920T000000, start_date=20240921T182055, end_date=20240921T182057
[2024-09-21T18:20:57.716+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-21T18:20:57.821+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-21T18:20:57.830+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
